woocommerce/woocommerce
# Extra i18n address handling for cart page compatibility

Fixes #16727 

To test:
Go to Cart page.
Select Liechtenstein in "Calculate shipping" form.
Observe placeholder changes.
Select Canada in "Calculate shipping" form.
Observe placeholder changes in select.
